config/index.js

```diff
@@ -155,7 +155,7 @@ var conf = convict({
     }
   },
   customsUrl: {
-    doc: 'fraud / abuse server url',
+    doc: 'fraud / abuse server url; set to the string \'null\' to disable',
```

Comment:
[memcached config uses `none` to disable](https://github.com/mozilla/fxa-auth-server/blob/524c35b88ed7bdc7e46537cf7e1775e4cfa1cc65/config/index.js#L55), we should stay consistent here and it looks like [`none` is the expected magic value](https://github.com/mozilla/fxa-auth-server/blob/524c35b88ed7bdc7e46537cf7e1775e4cfa1cc65/lib/customs.js#L38).

Reply:
It also looks like [`none` is set by default in dev.json](https://github.com/mozilla/fxa-auth-server/blob/524c35b88ed7bdc7e46537cf7e1775e4cfa1cc65/config/dev.json#L5), do we want to remove this for local development?